chain/consensus/mir/mempool/handlers/ids.go

```diff
@@ -0,0 +1,65 @@
+package handlers
+
+import (
+	"github.com/filecoin-project/lotus/chain/consensus/mir/mempool/types"
+	"github.com/filecoin-project/mir/pkg/dsl"
+	mpdsl "github.com/filecoin-project/mir/pkg/mempool/dsl"
+	"github.com/filecoin-project/mir/pkg/pb/mempoolpb"
+	"github.com/filecoin-project/mir/pkg/pb/requestpb"
+	"github.com/filecoin-project/mir/pkg/serializing"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+// IncludeComputationOfTransactionAndBatchIDs registers event handler for processing RequestTransactionIDs and
+// RequestBatchID events.
+func IncludeComputationOfTransactionAndBatchIDs(
```

Comment:
This file is copied from `SimpleMempool` without modification, right?

Reply:
Removed